pom.xml

```diff
@@ -224,6 +248,8 @@
         <mockito.api.version>2.0.7</mockito.api.version>
         <wiremock.version>3.8.0</wiremock.version>
         <eclipse.jetty.version>11.0.22</eclipse.jetty.version>
+    
+        <jacoco.version>0.8.6</jacoco.version>
```

Comment:
This version has lot of security issues present. Can we upgrade to latest?

Reply:
Fixed